src/cuga/sdk_core/debug_crm_finance_policy_e2e.py

```
@@ -0,0 +1,318 @@
+}
+
+
+async def create_and_process_policy(agent):
+    """Create policy and generate examples and guard code."""
+    print("\nStep 1: Creating and processing policy...")
+    print("="*60)
+    
+    print(f"\n--- Processing Policy: {POLICY_CONFIG['name']} ---")
+    
+    # Create policy
+    print(f"Creating policy...")
```

Ruff (0.15.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)